composer.json

```diff
@@ -11,9 +11,9 @@
     }],
     "require": {
         "php": ">=5.4",
-        "symfony/css-selector": "2.8.*",
+        "apache/log4php": "2.3.0",
         "facebook/php-sdk-v4": "~5.0",
-        "apache/log4php": "2.3.0"
+        "symfony/css-selector": "^3.0.4"
```

Comment:
Just a heads-up, always using the `--sort-packages` option, since it is a lot easier to spot items in a list when that list is sorted.

```
$ composer require --sort-packages symfony/css-selector:^3.0.4
```

For reference, see https://github.com/composer/composer/pull/3549.